### PR TITLE
Make functions pow and power aliases.

### PR DESCRIPTION
**Pull request by akshatnair**
Currently pow() was the name of the method supported
and this commit added pow as an alias and renamed the
method as power which is consistent with ANSI.

Testing: added unit test

### CODE REVIEW

**Comment by KBP-TDC** [comment]
please fix a type in your commit message ("conistent")

**Comment by akshatnair** [comment]
fixed typo.

File: presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java [hunk 1]
```diff
@@ -313,6 +313,33 @@ public void testPow()
         assertFunction("pow(NULL, NULL)", DOUBLE, null);
         assertFunction("pow(5.0, NULL)", DOUBLE, null);
         assertFunction("pow(NULL, 5.0)", DOUBLE, null);
+
```
Comment (losipiuk) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java]: I would extract private testPower(String methodAlias) 
and call it twice:
```java
testPower("pow");
testPower("power");
```
This way you will avoid copy pasting two blocks which differ only name of called scalar.

**Comment by losipiuk** [comment]
looks good % comment

File: presto-docs/src/main/sphinx/functions/math.rst [hunk 2]
```diff
@@ -80,6 +80,10 @@ Mathematical Functions
 
 .. function:: pow(x, p) -> double
 
+    Alias for :func:`power`
```
Comment (electrum) [inline presto-docs/src/main/sphinx/functions/math.rst]: Please use the same format as other aliases:
```
This is an alias for :func:`power`.

File: presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java (hunk 1 above)
Comment (electrum) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java]: I don't think we actually need all the tests for aliases. I think having a single assertion would be fine:
```java
// test alias
assertFunction("power(5, 3)", DOUBLE, 125.0);
assertFunction("pow(5, 3)", DOUBLE, 125.0);

Comment (electrum) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java]: (we have repeated tests for "ceiling", but could do the same there)

**Comment by akshatnair** [comment]
addressed comments

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**